providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/common/Convert.java

```diff
@@ -0,0 +1,184 @@
+package dev.openfeature.contrib.providers.flagd.resolver.common;
```

Comment:
Not a single method in this class is new; they were all extracted from [here](https://github.com/open-feature/java-sdk-contrib/pull/967/files#diff-e5ada6a457791691f43b3f36c1cea2708bcf029ea0d13a8e5c6044c133261f8aL203) to avoid duplication.